code/bmpman/bmpman.cpp

```diff
@@ -45,7 +46,20 @@
 /**
  * @todo upgrade this to an inline funciton, taking bitmap_entry and const char* as arguments
  */
-#define EFF_FILENAME_CHECK { if ( be->type == BM_TYPE_EFF ) strcpy_s( filename, be->info.ani.eff.filename ); else strcpy_s( filename, be->filename ); }
+#define EFF_FILENAME_CHECK \
```

Comment:
This macro is getting a bit too big, how about converting it to a static function?